src/modules/dashboards/dashboardsSaga.ts

```diff
@@ -693,6 +702,17 @@ export function* calculateYPositionAndAddWidget({
   yield put(addWidgetToDashboard(dashboardId, widgetId));
 }
 
+export function* resetDashboardFilters({
```

Comment:
Could You please, move it to the separate file under `saga` directory in dashboards module.
The `saveDashboard`, `cloneDashboard` are currently there and we are in the process of decomposing dashboardsSaga into smaller files.